src/moldybreadpkg/fedora.nim

```diff
@@ -365,13 +365,17 @@ method populate_results*(this: FedoraRequest): seq[string] {. base .} =
   while token.len > 0:
     try:
       stdout.write("->")
-      response = this.client.getContent(request)
-      new_pids = this.grab_pids(response)
-      for pid in new_pids:
-        result.add(pid)
-      token = this.get_token(response)
-      request = fmt"{base_request}&sessionToken={token}"
-      stdout.flushFile()
+      response = this.client.request(request, httpMethod = HttpGet)
+      if response.status == "200 OK":
+        notice(fmt"Successfully retrieved populate_results {request}.")
+        new_pids = this.grab_pids(response.body)
+        for pid in new_pids:
+          result.add(pid)
+        token = this.get_token(response.body)
+        request = fmt"{base_request}&sessionToken={token}"
+        stdout.flushFile()
```

Comment:
stdout.flushFile() is what keeps the `->` going on one line between `[` and `]`.

Just to be safe in case we get errors, let's put this after the error on 378, but outside of the if else block.  If not, we could end up with something wonky like:

[->->->->
->
->
->->->
->]